cmd/api.go

```diff
@@ -66,15 +67,28 @@ func resourceForFlags(f *flag.FlagSet) *resource.Resource {
 // APICmd represents the resource command
 func (o *apiOptions) runAddAPI() {
 	dieIfNoProject()
-
+	projectInfo, err := scaffold.LoadProjectFile("PROJECT")
+	if err != nil {
+		log.Fatalf("failed to read the PROJECT file: %v", err)
+	}
 	reader := bufio.NewReader(os.Stdin)
 	if !o.resourceFlag.Changed {
-		fmt.Println("Create Resource under api [y/n]?")
+		if projectInfo.Version == project.Version1 {
+			fmt.Println("Create Resource under pkg/apis [y/n]")
+		}
+		if projectInfo.Version == project.Version2 {
+			fmt.Println("Create Resource under api [y/n]")
+		}
 		o.apiScaffolder.DoResource = util.Yesno(reader)
 	}
 
 	if !o.controllerFlag.Changed {
-		fmt.Println("Create Controller under controllers [y/n]?")
+		if projectInfo.Version == project.Version1 {
+			fmt.Println("Create Controller under pkg/controller [y/n]")
+		}
+		if projectInfo.Version == project.Version2 {
+			fmt.Println("Create Controller under controllers [y/n]")
+		}
```

Comment:
I think we can take out file-path (pkg/apis or api, controllers or pkg/controller) from these prompts because we do print-out where resource/controllers source code is being scaffolded.

Reply:
how would we go ahead with the examples section?

Reply:
yes, we should update those to keep it in sync.